lib/normalize.js

```diff
@@ -34,6 +34,8 @@ module.exports = function normalize(nemo, _locator) {
   var normalizedLocator;
   if (_locator.constructor === String) {
     locator = _splitLocator(nemo, _locator);
+  } else if (locator instanceof nemo.wd.By.constructor) {
+      return locator; //no need to normalize, WebDriver understands this format.
   }
```

Comment:
oh the example I gave was poor. This use case originated when I wanted do something like `nemo.view._finds(nemo.view.simple.languageBy())`